hedera/tutorials/smart-contracts/hss-x-evm-part-1-schedule-smart-contract-calls.mdx

```diff
@@ -0,0 +1,851 @@
+---
+title: "HSS x EVM - Schedule Smart Contract Calls (Part 1)"
+---
+
+On most EVM chains like Ethereum, smart contracts cannot "wake up" on their own—every function call must be triggered by an externally owned account (EOA) or an off-chain bot. This means implementing time-based automation (like cron jobs) requires external infrastructure.
+
+**Hedera changes this fundamentally.**
+
+With the **[Hedera Schedule Service (HSS) via HIP-755](https://hips.hedera.com/hip/hip-755)** and **[HIP-1215](https://hips.hedera.com/hip/hip-1215)**, smart contracts on Hedera can schedule future calls to themselves or other contracts. The Hedera network itself stores and executes these scheduled transactions when the time comes—**no off-chain bots required**.
+
+This unlocks powerful new patterns:
+
+- **On-chain cron jobs** for DeFi rebalancing and automation
+- **Time-based vesting** and token releases
+- **Recurring payments** and subscriptions
+- **DAO governance** with time-delayed execution
+
+In this tutorial, you'll build a simple **AlarmClock** contract that demonstrates this unique capability by scheduling one-shot and recurring alarms entirely on-chain.
+
+<Info>
+  You can take a look at the **complete code** in the [**hss-schedule-sc-calls
+  demo
+  repository**](https://github.com/hedera-dev/hedera-code-snippets/tree/main/hss-schedule-sc-calls).
+</Info>
+
+---
+
+## What Makes This Unique?
+
+### Traditional EVM Chains (Ethereum, most L2s)
+
+- Smart contracts **cannot schedule** themselves to be called later
+- Every contract execution requires an external trigger (EOA or bot)
+- Time-based automation needs:
+  - Off-chain infrastructure (cron jobs, keepers, bots)
+  - Trusted third parties or decentralized keeper networks
+  - Additional gas costs for the keeper service
+
+### Hedera with HIP-1215
+
+- Contracts can call the **Schedule Service system contract** (`0x16b`)
+- Schedule any contract call to execute at a future time
+- The **Hedera consensus layer** stores and executes scheduled transactions
+- **Fully on-chain** automation with no external dependencies
+
+This is enabled by two key Hedera Improvement Proposals (HIPs):
+
+- **[HIP-755](https://hips.hedera.com/hip/hip-755)** – Schedule Service System Contract  
+  Exposes the Hedera Schedule Service as an EVM-callable system contract at address `0x16b`
+
+- **[HIP-1215](https://hips.hedera.com/hip/hip-1215)** – Generalized Scheduled Contract Calls  
+  Allows contracts to schedule **any contract call** via the Schedule Service, including calls to themselves
+
+---
+
+## Prerequisites
+
+- ECDSA account from the [Hedera Portal](https://portal.hedera.com/).
```

Comment:
I would consolidate prereq 1 and 3. Both require the portal. We can say `ECDSA account funded with minimum X amount of Hbar`?